pegasus/sites.v3/code.org/public/research/subgoal-labels-study/letter.md

```diff
@@ -145,11 +145,11 @@ Then the hard work of our research begins. We plan to share any and all findings
 
 Student work isn't lost, but you will have to directly go to the original URL to see it, and your teacher dashboard will only show progress for the subgoals version of Unit 3.
 
-Once you activate the subgoals version of Unit 3, it effectively switches you and any of your CSP students to using the subgoals version with a clean slate. Any student work in the original version of Unit 3 is preserved and students can see their work by entering the original Unit 3 URL directly: [studio.code.org/s/csp3](studio.code.org/s/csp3)
+Once you activate the subgoals version of Unit 3, it effectively switches you and any of your CSP students to using the subgoals version with a clean slate. Any student work in the original version of Unit 3 is preserved and students can see their work by entering the original Unit 3 URL directly: [studio.code.org/s/csp3-2019](studio.code.org/s/csp3-2019)
```

Comment:
I think this page is so old it should probably be deleted but if its not deleted I think the subgoal study it is referencing is from earlier than 2019. Baker is the person to ask about this.

Reply:
Good call. Yes, this is from 2017: https://github.com/code-dot-org/code-dot-org/blob/fe1396dc10c98965523cce66b883b706d79bb63a/dashboard/config/courses/csp-2017.course#L14-L18 I'll change it to use that version year for now.